R/format_MTA.R

```diff
@@ -0,0 +1,371 @@
+                  BroodSize_observed = as.integer(.data$BroodSize),
+                  BroodSize_min = .data$BroodSize_observed - .data$BroodSizeError,
+                  BroodSize_max = .data$BroodSize_observed + .data$BroodSizeError,
+                  #### Check if date of ringing may be considered as fledge date?
+                  FledgeDate_observed = as.integer(.data$RingagedNestlings),
+                  FledgeDate_min = NA_character_,
+                  FledgeDate_max = NA_character_,
+                  ## for new version of calc_clutchtype
+                  # NumberFledged_observed = as.integer(.data$NumberFledglings),
+                  NumberFledged = as.integer(.data$NumberFledged),
+                  NumberFledged_min = NA_integer_,
+                  NumberFledged_max = NA_integer_,
+                  AvgEggMass = NA_real_,
+                  NumberEggs = NA_integer_,
+                  AvgChickMass = NA_real_,
+                  NumberChicksMass = NA_integer_,
+                  AvgTarsus = NA_real_,
+                  NumberChicksTarsus = NA_integer_,
+                  #### Ask data owner for the tarsus method
+                  OriginalTarsusMethod = NA_character_,
+                  ExperimentID = if_else(.data$ExperimentId == "no_experiment",
+                                         NA_character_,
+                                         .data$ExperimentId)) %>%
+    dplyr::mutate(ClutchType_calculated = calc_clutchtype(data = ., na.rm = FALSE)) %>%
+    #### Rename
+    dplyr::rename(LayDate_observed = LayDate,
+                  NumberFledged_observed = NumberFledged) %>%
+    #### Final arrangement
+    dplyr::select(BroodID, PopID, BreedingSeason, Species, Plot, LocationID,
+                  FemaleID, MaleID,
+                  ClutchType_observed, ClutchType_calculated,
+                  LayDate_observed, LayDate_min, LayDate_max,
+                  ClutchSize_observed, ClutchSize_min, ClutchSize_max,
+                  HatchDate_observed, HatchDate_min, HatchDate_max,
+                  BroodSize_observed, BroodSize_min, BroodSize_max,
+                  FledgeDate_observed, FledgeDate_min, FledgeDate_max,
+                  NumberFledged_observed, NumberFledged_min, NumberFledged_max,
+                  AvgEggMass, NumberEggs, AvgChickMass, NumberChicksMass,
+                  AvgTarsus, NumberChicksTarsus, OriginalTarsusMethod, ExperimentID) %>%
+    distinct()
+
+  return(Brood_data)
+
+}
+
+
+#### CAPTURE DATA
+
+#' Create capture data table for blue tits in MTA-PE Evolutionary Ecology Group, Hungary.
+#'
+#' Create a capture data table in standard format for blue tits in MTA-PE Evolutionary Ecology Group, Hungary.
+#' @param Brood_data Data frame. Brood_data from MTA-PE Evolutionary Ecology Group, Hungary.
+#' @return A data frame.
+#'
+
+
+create_capture_MTA <- function(data) {
+
+  Capture_data <-
+    data %>%
+    dplyr::select(.data$Species, .data$PopID, .data$BreedingSeason,
```

Comment:
We still want to calculate species from the `species_code` table in case we want to change the codes in the future.

Reply:
In this data there is column called Species with "PARMAJ" value. Should I change it to extract the species from the `species_code `instead?

Reply:
Yes, that would be the best approach. In the future we might change the codes (especially if we start getting many different species from overseas), so we want to have it linked to the EURING code.